Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -765,6 +766,8 @@ TextPosition TextDocument::first_word_break_before(TextPosition const& position,
 
             target.set_column(*index);
             break;
+        } else {
+            break;
```

Comment:
it does not look bad in such form:
```suggestion
        }
        break;
```